### PR TITLE
Afsaneh/check migration

### PR DESCRIPTION
**Pull request by afsaneh-rezaeinejad-deriv**

### CODE REVIEW

File: cypress.config.js [hunk 1]
```diff
@@ -204,6 +204,27 @@ module.exports = defineConfig({
         ID: process.env.E2E_EU_LOGIN,
         PSWD: process.env.E2E_QA_ACCOUNT_PASSWORD,
       },
+      eligibleMigration1: {
+        ID: process.env.E2E_WALLETS_MIGARTION_MAIN,
+        PSWD: process.env.E2E_DERIV_PASSWOR
```
Comment (markodonnell-deriv) [inline cypress.config.js:209]: Missing the D in E2E_DERIV_PASSWOR

**Review by markodonnell-deriv — CHANGES_REQUESTED** [review]
typo

**Review by abuzar-deriv — APPROVED** [review]
LGTM

**Review by bilalmajid-deriv — APPROVED** [review]
LGTM

**Review by markodonnell-deriv — APPROVED** [review]
Looks good

**Pull request merged**